test262/per-file/index.html

```diff
@@ -0,0 +1,127 @@
+    />
+    <style></style>
+  </head>
+
+  <body>
+    <main>
+      <h1>LibJS test262 per-file result viewer</h1>
+      <p>
+        These are the per-file results of the SerenityOS JavaScript engine,
+        LibJS, running the official ECMAScript conformance test suite test262.
+        All tests are re-run and the results updated automatically for every
+        push to the master branch of the repository on GitHub.
+      </p>
+      <p>
+        The source code of the LibJS test262 runner and aggregation script is
+        available
+        <a
+          href="https://github.com/linusg/libjs-test262"
+          target="_blank"
+          rel="noopener noreferrer"
+          >here</a
+        >, the source code of this website
+        <a
+          href="https://github.com/linusg/libjs-website"
+          target="_blank"
+          rel="noopener noreferrer"
+          >here</a
+        >, and the raw data
+        <a
+          href="https://github.com/linusg/libjs-website/blob/main/test262/data/per-file-master.json"
+          target="_blank"
+          rel="noopener noreferrer"
+          >here</a
+        >
+        for the AST interpreter, and
+        <a
+          href="https://github.com/linusg/libjs-website/blob/main/test262/data/per-file-master-bytecode.json"
+          target="_blank"
+          rel="noopener noreferrer"
+          >here</a
+        >
+        for the bytecode interpreter. (JSON).
+      </p>
+      <p>
+        If you have any questions or want to help out with improving these test
+        scores, feel free to get in touch on the
+        <a
+          href="https://discord.gg/serenityos"
+          target="_blank"
+          rel="noopener noreferrer"
+          >SerenityOS Discord server</a
+        >
+        (<code>#js</code>).
+      </p>
+      <div>
+        <h2>Results</h2>
+        <div class="mode-summary-container">
+          <span id="summary">Loading...</span>
+          <div class="tree-node-status-container">
+            <span id="summary-status" class="hidden tree-node-status"></span>
+          </div>
+        </div>
+        <div style="height: 60px"></div>
+        <ul id="results"></ul>
+      </div>
+    </main>
+    <footer>
+      Made by
+      <a
+        href="https://twitter.com/linusgroh"
+        target="_blank"
+        rel="noopener noreferrer"
+        >@linusgroh</a
+      >
+      &
+      <a
+        href="https://twitter.com/the_semicolon_"
+        target="_blank"
+        rel="noopener noreferrer"
+        >CxByte</a
+      >.
+    </footer>
+    <script src="../test-config.js"></script>
+    <script src="main.js"></script>
+    <noscript
+      ><img
+        src="https://shynet.linus.dev/ingress/441b8324-3bdc-423d-b32b-5443edb43a26/pixel.gif"
+    /></noscript>
+    <script
+      defer
+      src="https://shynet.linus.dev/ingress/441b8324-3bdc-423d-b32b-5443edb43a26/script.js"
+    ></script>
+
+    <template id="leaf-tree-node-template">
+      <li id="node">
```

Comment:
IDs are [supposed to be unique](https://softwareengineering.stackexchange.com/a/127180) (browsers won't stop you from having duplicates on a page, it just doesn't make much sense). This should definitely be a class considering we're reusing this template.

Also, I've never ever used an HTML `<template>` before, cool stuff 👀

Reply:
:thonk: I see...alright, makes sense